### DISCUSSION TITLE
Static version for MacOS?

### DISCUSSION THREAD

**Discussion by speedgeil** [ImageMagick/ImageMagick#4804] [Help]
Hello everyone,
i had a little IM project on Windows going and when i was done i thought "Hey let's build a Hackintosh and do that again!".
After tinkering a week with Opencore i went over to imagemagick.org and wanted to download a static build for Monterey. Ha! How naive i am, right? 
You guess it, i'm no expert in MacOSX terminal commands (maybe not even in anything) and whenever i see code from it my eyes will cross...
Anyways, i don't see a proper guide for building a static version on Mac in the docs (maybe blindness) and i wonder if i could benefit from work someone has already done. Does there exist a script or so for building static versions? I would need RAW and TIF support in this new world of windows with rounded corners and smooth transitions...
Thanks for your time!

**Comment by speedgeil** [comment]
Ok, so i managed to compile a static version. Not sure if everything is correct.
Now i get this error when processing an image:
`magick: UnableToOpenBlob '/var/folders/77/brk9xq2s2112g6hz5k7bcpvr0000gn/T/magick-62474GjdJd1V3om8.ppm': No such file or directory @ error/blob.c/OpenBlob/3497.`
A problem with permissions seems quite obvious. I'm quite new to Macintosh/Unix ecosystem and i don't know how to solve this.
I'm on the newest version of Monterey.

**Comment by ivanooi** [comment]
Hi, which version of IM you compile? under which Mac version? is it 11? thanks . I managed to compile that but I'm having issues when move the folder to another Mac.

**Comment by bioworldmerch** [comment]
I know this is a dupe but wanted to let @ivanooi know the steps I had success with...see

https://github.com/ImageMagick/ImageMagick/discussions/5215

**Reply by ivanooi** [reply]
Thanks :-D
